+logging/logging.m

```diff
@@ -184,7 +184,7 @@ function writeLog(self, level, caller, message)
       end
 
       if self.logLevel_ <= level && self.logfid > -1
-        fprintf(self.logfid, logline);
+        fprintf(self.logfid, '%s', logline);

```

Comment:
Would the problem be solved if we removed the `if rows > 1` on line 251 in `getMessage()`?

Reply:
I was about to submit a pull request for the same change. A similar problem happens if there is a % in your output.